app/src/main/res/values-nl/strings-nl.xml

```diff
@@ -7,21 +7,21 @@
 	<string name="menu_stop">Stop</string>
 	<string name="menu_refresh">Verversen</string>
 	<string name="menu_export_database">Export databank</string>
-	<string name="menu_export_csv_sidiary">Export CSV (dochteronderneming-formaat)</string>
-	<string name="menu_import_db">Import-database</string>
+	<string name="menu_export_csv_sidiary">Exporteer CSV (dochteronderneming-formaat)</string>
```

Comment:
I would change this to "Exporteer CSV (SiDiary format)", as SiDiary doesn't really translate